src/components/Texts/Texts.ts

```diff
@@ -42,16 +42,18 @@ export const TitleBlack = styled.div`
   text-align: center;
 `;
 
-export const BlackText = styled.h2`
+export const BlackText = styled.h2<{ mt?: number; textCenter?: boolean; }>`
   font-family: 'Source Sans Pro';
   font-size: 0.875rem;
   line-height: 20px;
   color: ${colors.mineShaft};
-  text-align: left;
+  text-align: ${({ textCenter }) => (textCenter ? 'center' : 'left')};
   font-weight: 400;
   margin-left: auto;
   margin-right: auto;
   white-space: pre-wrap;
+  
+  ${({ mt }) => mt && `margin-top: ${mt}px !important`};
```

Comment:
Move `;` inside.
```suggestion
  ${({ mt }) => mt && `margin-top: ${mt}px !important;`}
```
As you are evaluating to add margin-top, if the result is false it will be shown as
```
white-space: pre-wrap;
;
padding-left: 20px;
```